k8s/airflow-operator/README.md

```diff
@@ -0,0 +1,157 @@
+# Overview
```

Comment:
This readme will require a line-up with other readmes after the tech writer's review (see #155).

Reply:
done. synced to cassandra readme.

Reply:
@barney-s thank you for this!